projects/aca-content/src/lib/ui/overrides/ay11.scss

```diff
@@ -157,13 +157,21 @@
     }
   }
 
+  .mat-datetimepicker-calendar-body-active .mat-datetimepicker-calendar-body-cell-content {
+    outline: 2px solid var(--theme-blue-button-color);
+  }
+
   .mat-datetimepicker-calendar-body-active .mat-datetimepicker-calendar-body-cell-content:not(.mat-datetimepicker-calendar-body-selected) {
     background-color: white !important;
```

Comment:
white should be moved to var as well

Reply:
Moved 'white' to its own CSS variable. New variable name is `$datetimepicker-cell-background-color` and `--theme-datetimepicker-cell-background`